admin.adoc

```diff
@@ -861,15 +861,12 @@ the following command line.
 grr_config_updater upload_python --file=myfile.py --platform=windows
 ------------------------------------------------------------------------
 
-At the end of this you should see something like:
-------------------------------------------------------------------------
-Uploaded successfully to aff4:/config/python_hacks/myfile.py
```

Comment:
It should be "Uploaded to aff4:/config/python_hacks/myfile.py" (per https://github.com/google/grr/blob/e4d6919be3ce631a123dfe7f4aa1a4769d7ec2e2/grr/lib/maintenance_utils.py#L92).
But apparently this message is not shown unless "--verbose" flag is provided. Probably fine to remove this part.